staging/src/k8s.io/kube-aggregator/pkg/controllers/openapiv3/aggregator/downloader_test.go

```diff
@@ -23,6 +23,7 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+	"k8s.io/kube-openapi/pkg/handler3"
```

Comment:
nit: import grouping

Reply:
@jiahuif I am not sure what this comment refers to. This ordering is determined by `gofmt`. Do you mean to I should add another newline? or could you show explicitly what you are looking for?

Reply:
Yes, the convention is
import (
standard libraries

third-party libraries

k8s.io dependencies
)

Reply:
```suggestion

	"k8s.io/kube-openapi/pkg/handler3"
```